src/graphql/queries/soknad.graphql

```diff
@@ -6,6 +6,7 @@ fragment Soknad on Soknad {
     fom
     tom
     lest
+    sendtDato
```

Comment:
For at det skal kunne deployes som en ikke-breaking kan ikke querien også oppdateres samtidig. Da vil brukere på den nye appen som går mot den gamle appen få kræsj.